### PR TITLE
chore: cleanup share new comment popup

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- [Cleanup](https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1713457405662439) the share new comment popup.
- Also fixed a 

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://chore-popup-cleanup.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/hooks/useShareComment.ts [hunk 1]
```diff
@@ -44,7 +30,7 @@ export function useShareComment(
             text: `${post.title}\n${shortUrl}$}`,
           });
           trackEvent(
-            postAnalyticsEvent('share post', post, {
+            postAnalyticsEvent('share comment', post, {
```
Comment (sshanzel) [inline packages/shared/src/hooks/useShareComment.ts:33]: @idoshamun this hook is for sharing a comment, but the analytics say `share post` - should we make this change to make it more accurate?

**Review by nensidosari — APPROVED** [review]
Love the cleanupss

**Comment by sshanzel** [comment]
> Love the cleanupss

Yup, me too. Especially when long prop/variable names are involved 😂

File: packages/shared/src/hooks/useShareComment.ts (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/hooks/useShareComment.ts:33]: Yep. Verified all usages to be only for comments.

**Pull request merged**